gui/gamesetup/gamesetup~!extra_commands.js

```diff
@@ -268,7 +268,8 @@ g_NetworkCommandsDescriptions = Object.assign(g_NetworkCommandsDescriptions,{
 	"/ban": "Ban player",
 	"/banspecs": "Ban all specs",
 	"/list": "List all the players and observers currently here",
-	"/clear": "Clear the chat comments"
+	"/clear": "Clear the chat comments",
+	  "/jitsi": "meet.jit.si/anyNameYoutWantHere"
```

Comment:
align spaces with others

Reply:
Bad description. Tell what is does, the use. 
example: "Create a game videocall. Uses jitsi (meet.jit.si) service"